src/models/postModel.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose"
+
+const postSchema = new mongoose.Schema({
+    title: {
+      type: String,
+      required: true,
+    },
+    content: {
+      type: String,
+      required: true,
+      
+    },
+    hashtags: [String],
+    comments:[{type: String,default:''}],
+    like: { type: Number, default: 0 },
+    author: [String],
+    authorId: { type: String, required: true }
+  }, {timestamps: true});
+  
+  export const postmodel = mongoose.model("post_collection", postSchema);
```

Comment:
First you create an interface for the model 
`interface IPost {
title: string;
  content: string;
}

export default mongoose.model<IPost> = mongoose.model("user", postSchema)
`